prowler/providers/aws/services/securityhub/securityhub_service.py

```diff
@@ -32,67 +34,73 @@ def __threading_call__(self, call):
                     is_resource_filtered(hub_arn, self.audit_resources)
                 ):
                     hub_id = hub_arn.split("/")[1]
+                    get_enabled_standards_paginator = regional_client.get_paginator(
+                        "get_enabled_standards"
+                    )
+                    standards = ""
+                    for page in get_enabled_standards_paginator.paginate():
+                        for standard in page["StandardsSubscriptions"]:
+                            standards += f"{standard['StandardsArn'].split('/')[1]} "
+                    list_enabled_products_for_import_paginator = (
+                        regional_client.get_paginator(
+                            "list_enabled_products_for_import"
+                        )
+                    )
+                    integrations = ""
+                    for page in list_enabled_products_for_import_paginator.paginate():
+                        for integration in page["ProductSubscriptions"]:
+                            if (
+                                "/aws/securityhub" not in integration
+                            ):  # ignore Security Hub integration with itself
+                                integrations += f"{integration.split('/')[-1]} "
                     self.securityhubs.append(
                         SecurityHubHub(
                             hub_arn,
                             hub_id,
                             "ACTIVE",
                             standards,
+                            integrations,
                             regional_client.region,
                         )
                     )
                 else:
+                    # SecurityHub is filtered
                     self.securityhubs.append(
                         SecurityHubHub(
                             "",
                             "Security Hub",
                             "NOT_AVAILABLE",
                             "",
+                            "",
                             regional_client.region,
                         )
                     )
 
         except Exception as error:
-            # Check if Account is subscribed to Security Hub
-            if "InvalidAccessException" in str(error):
-                self.securityhubs.append(
-                    SecurityHubHub(
-                        "",
-                        "Security Hub",
-                        "NOT_AVAILABLE",
-                        "",
-                        regional_client.region,
-                    )
-                )
-            else:
-                logger.error(
-                    f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
-                )
+            logger.error(
+                f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
 
 
 @dataclass
```

Comment:
Can you use the Pydantic `BaseModel` here? We have to migrate all the classes we have.

Reply:
Done